docs/rst/platforms/platforms.rst

```diff
@@ -15,6 +15,6 @@ Vulcanexus distribution is officially available in the platforms specified in th
       - amd64
       - Ubuntu Focal (20.04)
 
-However, Vulcanexus is an extended ROS 2 distribution which is officially supported in the platforms stated in the `REP 2000 specification <https://www.ros.org/reps/rep-2000.html>`_.
-Thus, building Vulcanexus distribution for these platforms is expected to succeed.
+Vulcanexus ROS 2 all-in-one tool set is officially supported in the platforms stated in the `REP 2000 specification <https://www.ros.org/reps/rep-2000.html>`_.
```

Comment:
The previous meant that even though Vulcanexus official support is limited to Ubuntu Focal, it is still a ROS 2 extension, and ROS 2 has official support in all of the platforms specified in the REP 2000. This means that Vulcanexus is expected to work perfectly in all those platforms, since ROS 2 does.

The proposed changes do not reflect this and in fact contradict the information on the previous table 😖